src/qudi/hardware/ni_x_series/ni_x_series_finite_sampling_input.py

```diff
@@ -592,7 +594,7 @@ def _init_digital_tasks(self):
                         min_val=0,
                         max_val=100000000,
                         units=ni.constants.TimeUnits.TICKS,
-                        edge=ni.constants.Edge.RISING)
+                        edge=ni.constants.Edge.FALLING if self._invert_trigger_polarity else ni.constants.Edge.RISING)
```

Comment:
With the constructor, I think this will simply be `edge=self._trigger_polarity` ...